### PR TITLE
Configurable params for SC address

### PR DESCRIPTION
**Pull request by evgeny-s**

### CODE REVIEW

File: frame/ddc-metrics-offchain-worker/src/lib.rs [hunk 1]
```diff
@@ -102,6 +102,20 @@ impl MetricInfo {
     }
 }
 
+pub fn get_contract_id() -> [u8; 32] {
+	let sc_address_store = StorageValueRef::persistent(b"ddc-metrics-offchain-worker::sc_address");
```
Comment (naure) [inline frame/ddc-metrics-offchain-worker/src/lib.rs]: Are we able to set this value from the UI or CLI?

**Review by naure — APPROVED** [review]

File: frame/ddc-metrics-offchain-worker/src/lib.rs (hunk 1 above)
Comment (evgeny-s) [reply frame/ddc-metrics-offchain-worker/src/lib.rs]: Yes, sure

Comment (naure) [reply frame/ddc-metrics-offchain-worker/src/lib.rs]: What is the format in the block viewer?

Contract address must decode to `[u8; 32]` (fixed-size array)
DDC URL must decode to `Vec<u8>` (UTF-8 string as bytes)

Decoding using [CODEC](https://substrate.dev/docs/en/knowledgebase/advanced/codec#vectors-lists-series-sets).

Comment (evgeny-s) [reply frame/ddc-metrics-offchain-worker/src/lib.rs]: Let me check

Comment (evgeny-s) [reply frame/ddc-metrics-offchain-worker/src/lib.rs]: Seems like block viewer sends it as byte array

Comment (evgeny-s) [reply frame/ddc-metrics-offchain-worker/src/lib.rs]: @naure 
Will merge it as is. At least we can configure SC address.

**Pull request merged**